TailLight/device.h

```diff
@@ -1,13 +1,37 @@
 #pragma once
 
+#include "debug.h"
+
+#define MSINTELLIMOUSE_USBINTERFACE5_PREFIX L"\\??\\HID#VID_045E&PID_082A&MI_01&Col05"
+
+#define BEGIN_WITH(x) { \
+        auto &_ = x;
+#define END_WITH() }
```

Comment:
What are these defines used for?

Reply:
This was future work for dealing with members of a structure in a row, in order to avoid "I Love to Type" (ILT). Other languages have a With() statement, like:

BEGIN_WITH(GPIO_PORT_CONTROL)
    _.AFSEL = 0
    _.DIGITAL_ENABLE = 1
    _.PIN_ENABLE = 0x5
END_WITH

However, the final version that I'm going with won't need to work with members of a structure or class in succession, so I removed it. Again, learning the checkin rules of the organization.

Reply:
Please remove unrelated changes from this pull request. I only want to merge the changes needed to fix the reported problem.